install.sh

```diff
@@ -76,6 +77,69 @@ function backup_append() {
 	echo -e "\033[0;32mBash-it template has been added to your $CONFIG_FILE\033[0m"
 }
 
+function check_for_backup() {
+	if ! [ -e "$HOME/$BACKUP_FILE" ]; then
```

Comment:
double bracket test are technically safer. if we're writing or transposing new things, I'd like to push to switch

Reply:
done